packages/plugin-conversation/src/conversation.js

```diff
@@ -101,6 +124,28 @@ const Conversation = SparkPlugin.extend({
       .then(() => res.body));
   },
 
+  leave(conversation, object, activity) {
+    this._inferConversationUrl(conversation);
```

Comment:
I guess my comment was a long drawn out way of saying - why aren't we calling treating this call to _inferConversationUrl like every other one and handling the promise it returns?